target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -679,6 +679,12 @@ tl-wr941nd-v6)
 	ucidef_set_led_switch "lan4" "LAN4" "tp-link:blue:lan4" "switch0" "0x02"
 	ucidef_set_led_wlan "wlan" "WLAN" "tp-link:blue:wlan" "phy0tpt"
 	;;
+tl-wr840n-v2|\
+tl-wr840n-v3)
+	ucidef_set_led_netdev "wan" "WAN" "tp-link:green:wan" "eth1"
+	ucidef_set_led_netdev "lan" "LAN" "tp-link:green:lan" "eth0"
```

Comment:
No problem, I'm going to fix it. 
But it looks like it works someway - the ethernet switch replicates LAN ports link states to the port link state.
When all external LAN ports are diconnected, the eth0 port has link down state:

root@LEDE:~# dmesg
...
[  236.689696] eth0: link down
[  236.694266] br-lan: port 1(eth0.1) entered disabled state
[  373.189175] eth0: link up (1000Mbps/Full duplex)

Even if br-lan is UP.